pre-commit/web_lint_check.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import multiprocessing
+import sys
+
+
+def web_lint_check():
+    if len(sys.argv) <= 2:
+        print("At least 2 arguments are required")
+        print("usage: python web_lint_check.py [js | css | misc] [<file_paths>]")
+        exit(2)
```

Comment:
No argparse? :)
https://docs.python.org/3/library/argparse.html

Reply:
ah that's probably better